app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -445,6 +466,10 @@ class ComposeViewModel
         scheduledAt.value = newScheduledAt
     }
 
+    private fun isEditingScheduledToot(): Boolean {
+        return !scheduledTootUid.isNullOrEmpty()
+    }
+
```

Comment:
As per Kotlin styleguide https://kotlinlang.org/docs/reference/coding-conventions.html#functions-vs-properties one could make this a property

```suggestion
    private val isEditingScheduledToot get() = !scheduledTootUid.isNullOrEmpty()
```

Reply:
Fair point. Fixed with a4b20b5